bundlesize.config.json

```diff
@@ -21,107 +21,107 @@
       "maxSize": "23 kB"
     },
     {
-      "path": "public/packs/analyticsArticle-*.js",
+      "path": "public/packs*/analyticsArticle-*.js",
```

Comment:
Ah, so that's why it was failing. I tested it locally and seems to be working well!